src/go.mod

```diff
@@ -1,6 +1,6 @@
 module code.cloudfoundry.org/loggregator-agent
 
-go 1.12
```

Comment:
I'm not sure if we should be changing the minimum go version during backports.. is this necessary to the bump? Or could we take it out?

Reply:
I thought the optics were weird here with the minimum go version fluctuating so much between versions. It's been pointed out that this shouldn't introduce any problems though.